docs/pages/Manifesto.md

```diff
@@ -0,0 +1,190 @@
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+[1]https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+-->
+
+# Adoptium Quality Assurance (AQA) Definition v1.0
+
+### Table of contents
+
+1. [Introduction](#introduction)
+2. [Guidelines for AQA](#guidelines)
+    1. [Open and Transparent](#openTransparent)
+    2. [Diverse and Robust](#diverseRobust)
+        1. [Functional and Regression Tests](#functionalRegression)
+        2. [External Application Tests](#externalApp)
+        3. [Performance Benchmarks](#performance)
+        4. [Security Tests](#security)
+    3. [Evolution alongside Implementations](#evolution)
+        1. [Continual Investment](#continualInvest)
+        2. [Process to Modify](#processToModify)
+        3. [Codecov and other Metrics](#metrics)
+        4. [Comparative Analysis](#comparativeAnalysis)
+    4. [Portable](#portable)
+    5. [Tag and Publish](#tagPublish)
+3. [Summary](#summary)
+
+<a name="introduction"></a>
+
+## 1 Introduction
+
+At the Adoptium project, we are committed to producing high-quality OpenJDK binaries, which we call Eclipse Temurin, for consumption by the community.  After speaking with different implementers, and listening to the needs of our community, developers and enterprise consumers alike, we have heard very clearly the desire to guarantee a certain level of quality.  We organise this quality assurance work within the Eclipse AQAvit project, which is named by the Adoptium Quality Assurance (AQA) acronym, in combination with 'vit' and its dual meaning for speed and vitality.
+
+Quality Assurance means, “Make quality certain to happen”.  Our goal is to make it certain that the project delivers high-quality binaries, that satisfy enterprise-grade requirements.  Enterprises and developers alike require that the binaries be:
+- Secure – ensure binaries are not susceptible to known security vulnerabilities
+- Functional – runs the apps they care about, on platforms they care about
+- Performant – meets or exceeds a set performance bar for various types of operations
+- Scalable – can span and scale to enterprise workloads
+
+<a name="guidelines"></a>
+
+## 2 Guidelines for AQA
+
+<a name="openTransparent"></a>
+
+### 2.1 Open & Transparent
+We believe open languages deserve open tests.  This means test source should be open and tests should be executed in the open with their results openly available.
+
+Transparent processes strengthen confidence.  Consumers get to see test results directly and know a certain quality bar was achieved, rather than just be told some testing was done.  Open tests get scrutinized, get fixed, get loved.
```

Comment:
I needed to google 'scrutinized'. Maybe we can use a more common phrase. Adding @ShelleyLambert